README.md

```diff
@@ -25,14 +25,6 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
```

Comment:
![Screenshot from 2023-02-16 01-13-55](https://user-images.githubusercontent.com/83042742/219213808-38122889-bff4-47c3-b5ab-413543fcd6b8.png)
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you change the above `footer` tag to a `div` tag or **even remove it** as you are still in the home page of your app and not at the `footer` yet. This for good semantic reasons.

Reply:
done